azurerm_frontdoor_custom_https_configuration key vault certificate source: do not require version (#11310)
terraform-providers/terraform-provider-azurerm: website/docs/r/frontdoor_custom_https_configuration.html.markdown

* `azure_key_vault_certificate_secret_name` - (Required) The name of the Key Vault secret representing the full certificate PFX.

> * `azure_key_vault_certificate_secret_version` - (Optional) The version of the Key Vault secret representing the full certificate PFX. If not given, the Latest version will be used.

Reviewer: ```suggestion
* `azure_key_vault_certificate_secret_version` - (Optional) The version of the Key Vault secret representing the full certificate PFX. Defaults to `Latest`.
```

Reviewer 2: may be too long, but what about:
```suggestion
* `azure_key_vault_certificate_secret_version` - (Optional) The version of the Key Vault secret. Defaults to 'Latest'. This will automatically be updated by Azure as new versions become available in the Key Vault, (it may take up to 24 hours to update).
```

Author: I went with @WodansSon's suggestion. IMO the Azure documentation should specify the Latest behavior, not necessarily the terraform documentation.
